test/functionalTest/cases/0501_cors/get_specific_origin.test

```diff
@@ -1,211 +0,0 @@
-# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
It will be great not losing old .test in case/0501 in order not losing "covegare" for the basic CORS feature (i.e. only for GET withoug pre-fligth) that works for NGSIv1.

Is there something in this test that is precluding to work with the new implementation?

Reply:
No but, the new logic covers that functionality. 

If the request is not a preflight and CORS is enabled, what we do now basically is the same thing. Adding the Allowed-Origin header to the response. And this functionality is covered in the new tests. 

The main reason I removed the tests was, it was only focusing on the GET method but in reality, simple CORS requests include POST and HEAD methods as well.

Reply:
> And this is covered in the new tests.

I understand new tests (in this PR and in the following ones you could have "queued" after it) cover the NGSIv2 URLs, but not NGSIv1 URL. Thus, until we can "cover" NGSIv1 with full CORS support (if at the end we decide that is important, which is something yet unders discussion on https://github.com/telefonicaid/fiware-orion/issues/501#issuecomment-333568837) I think is valuable to keep these tests.

However, maybe a FIXME like this could be added to the NGSIv1 based .test preamble:

```
FIXME: this .test would be reviewed (or maybe removed) once full CORS support gets completed for NGSIv1 URLs
```

Reply:
As a side note, in general, existing .test are not removed. Each single .test is a piece of the regression that ensures that, PR after PR, the code doesn't break backward compatiblity. Removing .test "decapitalize" our regression, i.e. makes regression weaker.

This is not a rule written in stone :), but make us very conservative in order to take .test removal decissions. Only in the case a new feature clearly overcomes an existing one, removal has been done. In case of doubt, we don't remove.

Reply:
I see, sure I will add them back.

Reply:
Fixed in 543c64b